generators/app/templates/src/app/screens/Login/layout.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18next from 'i18next';
+
+import InputLabel from '../../components/InputLabel';
+
+import { FIELDS } from './constants';
+import styles from './styles.scss';
+
+function Login({ handleChange, handleSubmit }) {
+  return (
+    <div className={`column center middle full-width ${styles.loginContainer}`}>
+      <form className={`column center space-between ${styles.formContainer}`} onSubmit={handleSubmit}>
+        <div className="column center full-width">
+          <h2 className="m-bottom-1">{i18next.t('Login:login')}</h2>
+          <h2>{i18next.t('Login:loginExplanation')}</h2>
```

Comment:
It's weird to have 2 `h2`. I think this second one should be mor of an `h3` to declare that it's a "subtitle"